lib/home/views/home_screen.dart

```diff
@@ -195,6 +256,29 @@ class _DeviceContent extends ConsumerWidget {
                 .titleSmall
                 ?.apply(color: Theme.of(context).colorScheme.onSurfaceVariant),
           ),
+        if (deviceData.info.pinComplexity)
+          Padding(
+            padding: const EdgeInsets.only(top: 12),
+            child: RichText(
```

Comment:
why not use just a wrap or row?

Reply:
Verical alignment between the icon and text is more "correct" when done this way.